test/extensions/filters/network/redis_proxy/proxy_filter_test.cc

```diff
@@ -38,6 +38,12 @@ parseProtoFromJson(const std::string& json_string) {
   envoy::config::filter::network::redis_proxy::v2::RedisProxy config;
   auto json_object_ptr = Json::Factory::loadFromString(json_string);
   Config::FilterJson::translateRedisProxy(*json_object_ptr, config);
+  return config;
+}
+
+void addV2ProtoFromJson(const std::string& json_string,
```

Comment:
We are killing v1. Can you just switch all the tests over to v2 config and delete all v1 code from this test? cc @derekargueta

Reply:
I'm pretty sure that the only V1 code is translateRedisProxy(), so I'll remove that. I'm going to directly call some of the JSON utility macros, but they all appear to be v2 macros.